Simplenote/Classes/SPNoteEditorViewController.m

```diff
@@ -567,30 +392,19 @@ - (void)prepareToPopView
                                             atScrollPosition:UITableViewScrollPositionTop
                                                     animated:NO];
     }
-    
 }
 
-
 - (void)backButtonAction:(id)sender
```

Comment:
`backButtonAction` is not triggered by back button. I see that it is used when deleting note from options or when note is deleted on another device. Also from the "pop" side gesture.
So maybe it makes sense to rename it to `dismiss` or something along those lines.